src/stacks/route53_acm_ssl_stack.ts

```diff
@@ -0,0 +1,47 @@
+// This is better to be in its own stack. 
```

Comment:
I don't think this is necessary as part of a library - maybe as a example of how you can construct these parts to setup a full stack.